src/quartz/schemas/Organization.yml

```
@@ -8,6 +8,9 @@ properties:
     description: URL for talking to a specific cluster
     readOnly: true
     type: string
+  orgName:
```

```suggestion
  name:
```

We're inside an `Organization` schema, so the `org` prefix is redundant.